tests/resources/Ohara_Rudy_2011.cellml

```diff
@@ -61,28 +61,28 @@ http://www.cellml.org/
     <units name="microF">
         <unit prefix="micro" units="farad"/>
     </units>
-    <units name="micrometer_3">
+    <units name="micrometre_3">
```

Comment:
We didn't need to change these (and others like it) as they are user defined.  The user is free to use whichever spelling they prefer.

Reply:
Agreed (I saw your previous comment about this above), it's just that the name of some units had already been modified (to have "litre" rather than "liter"), so I thought I would just do the same for metre. However, since I agree with your point, I am going to revert the name of all those variables. We indeed only need the value of `units` to use the correct spelling.